### PR TITLE
Compile only the needed sass file and use it

### PR DESCRIPTION
**Pull request by b2l**
Just compile mnd-bootstrap.scss to dist/mnd-bootstrap.css
Use the "compressed" output, which mean minify basically.
As we agreed on during the frontend chapter, we only expose the generated mnd-bootstrap.css and _variables.scss.

### CODE REVIEW

**Comment by dbackeus** [comment]
Maybe we shouldn't minify so early in the pipeline?

Rails will automatically minify in production env. But in dev env we could enjoy having readable css when debugging in browser etc.

File: doc_assets/_header.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <title>Mynewsdesk Style Guide: <%= title %></title>
-    <link rel="stylesheet" href="dist/mnd-bootstrap-dev.css">
+    <link rel="stylesheet" href="dist/mnd-bootstrap.css">
```
Comment (jorkas) [inline doc_assets/_header.html]: @b2l shouldn't this be "dist/mnd-bootstrap.min.css"

**Comment by b2l** [comment]
At the moment, we still retrieve the sourcemap, so it's fine for debugging.
We plan on changing the build process a bit, so we can document React components too.
I'll keep that in mind at that time

File: doc_assets/_header.html (hunk 1 above)
Comment (b2l) [inline doc_assets/_header.html]: Actually no. The build process with gulp doesn't allow me to rename the output file.
So it's the name of the base file (mnd-bootstrap), they just change the extension from .scss to .css

Comment (jorkas) [inline doc_assets/_header.html]: :+1:

**Pull request merged**